Lib/gettext.py

```diff
@@ -46,7 +46,6 @@
 #   find this format documented anywhere.
 
 
-import locale
 import os
```

Comment:
Please write a separated PR for gettext, and keep this PR for test.support.

Reply:
OK, I created #19905  to do this job.